src/canopy_cli/cli.py

```diff
@@ -210,6 +211,40 @@ def health(url):
     return
 
 
+@cli.command(help="Writes a config template YAML file to the specified path.")
+@click.argument("path", type=click.Path(), required=True)
+@click.option("--template", default="default", help="The name of the template to use.")
+def create_config(path, template):
+
+    if not template.endswith('.yaml'):
+        template += '.yaml'
+
+    if os.path.exists(path):
+        click.confirm(click.style(f"File {path} already exists. Overwrite?", fg="red"),
+                      abort=True)
+
+    try:
+        with pkg_resources.open_text('canopy.config', template) as f:
+            config = yaml.safe_load(f)
```

Comment:
@acatav you don't need this tedious loading and saving code. Just copy the entire content of the `config/` dir from the installed location to the user's desired path